src/main/java/com/github/splendor_mobile_game/websocket/handlers/reactions/MakeReservationFromDeck.java

```diff
@@ -239,7 +239,7 @@ private void validateData(DataDTO dataDTO, Database database) throws InvalidUUID
 
             
         // Check if user has not maxed tokens
-        if (user.getTokenCount() >= 10)
+        if (user.getTokenCount() > 10)
             throw new TokenCountException("You have reached the maximum token count on hand.");
```

Comment:
It won't ever be true.

Reply:
i know i know but i thought maybe if it happens to be true we will be notified